test/unit/logic/peers.js

```diff
@@ -105,7 +105,7 @@ describe('peers', function () {
 			});
 		});
 
-		it('should should not contain rpc when normalized', function () {
+		it('should not contain rpc when normalized', function () {
```

Comment:
To takes things further. These `when normalised` conditions could be better described as:

```js
describe('list', function () {

  describe('when normalised', function () {

    it('should contain peers as objects');

    it('should not have any rpc properties');
  });
});
```